### ISSUE TITLE
Caddy segfault on file-server subcommand

### ISSUE THREAD

**Issue by b12f** [caddyserver/caddy#5068]
Any command in the `caddy file-server` subcommand segfaults for me in the official docker container. `caddy file-server --help` works fine, but other than that it doesn't matter whether you use `--root` or `--listen` or anything else, it segfaults identically.

```
❯ docker run caddy:2-alpine caddy file-server
Unable to find image 'caddy:2-alpine' locally
2-alpine: Pulling from library/caddy
213ec9aee27d: Pull complete
fd0c7d01ba8a: Pull complete
e56da4be4f98: Pull complete
cc800527c197: Pull complete
b914d85a1ddd: Pull complete
Digest: sha256:eb2faa697dcfeaa8fe36edf57dab9d98d4d6cbd3f2eb541ada2ceb713ffec219
Status: Downloaded newer image for caddy:2-alpine
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xae7688]

goroutine 1 [running]:
github.com/caddyserver/caddy/v2/cmd.Flags.String({0x1841740?}, {0x1a7645e?, 0x1a7c02a?})
	github.com/caddyserver/caddy/v2@v2.6.0/cmd/main.go:267 +0x48
github.com/caddyserver/caddy/v2/cmd.Flags.Bool({0x1821540?}, {0x1a7645e?, 0xc00004c168?})
	github.com/caddyserver/caddy/v2@v2.6.0/cmd/main.go:275 +0x25
github.com/caddyserver/caddy/v2/modules/caddyhttp/fileserver.cmdFileServer({0x0?})
	github.com/caddyserver/caddy/v2@v2.6.0/modules/caddyhttp/fileserver/command.go:74 +0x125
github.com/caddyserver/caddy/v2/cmd.caddyCmdToCoral.func1(0xc000711b80?, {0x1a734ec?, 0x0?, 0x0?})
	github.com/caddyserver/caddy/v2@v2.6.0/cmd/cobra.go:114 +0x2c
github.com/spf13/cobra.(*Command).execute(0xc000711b80, {0x2ad0e50, 0x0, 0x0})
	github.com/spf13/cobra@v1.1.3/command.go:852 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x2a77a60)
	github.com/spf13/cobra@v1.1.3/command.go:960 +0x39d
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.1.3/command.go:897
github.com/caddyserver/caddy/v2/cmd.Main()
	github.com/caddyserver/caddy/v2@v2.6.0/cmd/main.go:64 +0x5c
main.main()
	./main.go:39 +0x17
```

This is not docker specific, I was able to reproduce this on NixOS.

```
❯ docker run caddy:2-alpine caddy version
v2.6.0 h1:lHDynvM+sTOi9Aq4Y15b4FtkqzPB36WbUrZvVdwzTCA=
```

**Comment by mohammed90** [comment]
This issues was fixed in v2.6.1, whose Docker image release is pending on Docker side: https://github.com/docker-library/official-images/pull/13204. It's unfortunately partly manual process.

**Comment by francislavoie** [comment]
Duplicate of https://github.com/caddyserver/caddy/issues/5058
